WooCommerce/Classes/POS/Presentation/SimpleProductCardView.swift

```diff
@@ -13,21 +13,11 @@ struct SimpleProductCardView: View {
 
     var body: some View {
         HStack(spacing: Constants.cardSpacing) {
-            if let imageSource = product.productImageSource {
-                ProductImageThumbnail(productImageURL: URL(string: imageSource),
-                                      productImageSize: Constants.productCardSize * scale,
-                                      scale: scale,
-                                      foregroundColor: .clear,
-                                      cachesOriginalImage: true)
-                .frame(width: min(Constants.productCardSize * scale, Constants.maximumProductCardSize),
-                       height: Constants.productCardSize * scale)
-                .clipped()
-            } else {
-                Rectangle()
-                    .frame(width: min(Constants.productCardSize * scale, Constants.maximumProductCardSize),
-                           height: Constants.productCardSize * scale)
-                    .foregroundColor(Color(.secondarySystemFill))
-            }
+            POSItemImageView(imageSource: product.productImageSource,
+                             imageSize: Constants.productCardSize * scale,
+                             scale: scale)
+            .frame(width: min(Constants.productCardSize * scale, Constants.maximumProductCardSize),
+                   height: Constants.productCardSize * scale)
```

Comment:
```suggestion
                   height: Constants.productCardSize * scale)
               .clipped()
```

I think we need to re-add clipped to limit the growth of images for extremely large accessibility sizes:

| With Clipped | Without Clipped |
|-------------|-----------------|
| ![with clipped](https://github.com/user-attachments/assets/66d68cfb-8a02-48df-9917-08fe2690ec74) | ![without clipped](https://github.com/user-attachments/assets/27d189a2-60b5-45cd-bcf9-1d4f74f3f518) |

For `ParentProductCardView` as well.

Reply:
Great catch, `clipped` needs to be placed after the `frame`. I think it'd be great to remove the if/else condition in the product image view and use a placeholder image instead of setting up the view differently based on the image source.

Reply:
IMO the ideal UX is to scale to fit with the same aspect ratio, I checked a few product images and they are cropped too much. I compared it with trunk, the cropping behavior is the same but it's out of the scope of this PR. I created a separate issue for this: https://github.com/woocommerce/woocommerce-ios/issues/14776.

<img src="https://github.com/user-attachments/assets/857384c1-2f53-4969-aa8d-685a5a965c06" width="400" />

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/14771/commits/093b9520275b549afbd3d0c4af7a053bead2860d.

Reply:
> I created a separate issue for this: https://github.com/woocommerce/woocommerce-ios/issues/14776.

Thank you!